rivalcfg/devices/__init__.py

```diff
@@ -250,21 +221,50 @@ def _generate_profiles():
 
     :rtype: dict
     """
+    from . import (  # noqa: F401
+        aerox3,  # noqa: F401
+        aerox3_wireless_wired,  # noqa: F401
+        aerox3_wireless_wireless,  # noqa: F401
+        aerox5_wireless_wired,  # noqa: F401
+        aerox5_wireless_wireless,  # noqa: F401
+        aerox9_wireless_wired,  # noqa: F401
+        aerox9_wireless_wireless,  # noqa: F401
+        kanav2,  # noqa: F401
+        kinzuv2,  # noqa: F401
+        prime,  # noqa: F401
+        prime_wireless_wired,  # noqa: F401
+        prime_wireless_wireless,  # noqa: F401
+        rival3,  # noqa: F401
+        rival3_wireless,  # noqa: F401
+        rival95,  # noqa: F401
+        rival100,  # noqa: F401
+        rival110,  # noqa: F401
+        rival300,  # noqa: F401
+        rival300s,  # noqa: F401
+        rival310,  # noqa: F401
+        rival500,  # noqa: F401
+        rival600,  # noqa: F401
+        rival650,  # noqa: F401
+        rival700,  # noqa: F401
+        sensei310,  # noqa: F401
+        sensei_raw,  # noqa: F401
+        sensei_ten,  # noqa: F401
+    )  # noqa: F401
+
+    profile_modules = locals()
     profiles = {}
-    for item in [globals()[name] for name in globals()]:
-        if not isinstance(item, types.ModuleType):
-            continue
-        if not hasattr(item, "profile"):
-            continue
+    for item in profile_modules.values():
         for model in item.profile["models"]:
             profile = item.profile.copy()
             profile_name = (model["vendor_id"], model["product_id"])
             del profile["models"]
             for k, v in model.items():
-                if k == "override_defaults":
-                    continue
                 profile[k] = v
-            # TODO override_defaults
+            #   if k == "override_defaults":
+            #            continue
+            #        profile[k] = v
+
+            #  TO-DO override defaults
```

Comment:
I prefere TODO than TO-DO as it is highlighted in my editor :)

![image](https://github.com/flozz/rivalcfg/assets/971438/a3e1287f-dc95-46aa-b1c2-c1ef61a974ec) vs 
![image](https://github.com/flozz/rivalcfg/assets/971438/f6e85ebc-7b8e-48a5-a141-0074c0f1d5dd)

anyway I think we should remove lines 263-267 and I should remove the related documentation (lines 34-37) as this was never implemented...

```suggestion
```